install/bootstrap_livecd_remote_iso.md

```diff
@@ -224,343 +246,280 @@ On first login (over SSH or at local console) the LiveCD will prompt the adminis
-            ```
+   > **`NOTE`** if it is desirable to expedite booting into the USB, this step may be skipped. Instead, after logging into the PIT for [#first time](#51-first-login) and running `pit-init` the USB will gain connectivity and SSH. As long as the required files are present then one may continue to the first time boot.
 
-            * If a cooling door is found in `hmn_connections.json`, there may be a message like the following. It can be safely ignored.
+      ```bash
+      pit# ls -1
+      ```
 
-               ```json
-               {"level":"warn","ts":1612552159.2962296,"msg":"Cooling door found, but xname does not yet exist for cooling doors!","row":
-               {"Source":"x3000door-Motiv","SourceRack":"x3000","SourceLocation":" ","DestinationRack":"x3000","DestinationLocation":"u36","DestinationPort":"j27"}}
+   Expected output looks similar to the following:
 
-      1. Skip the next step and continue with the [CSI Workarounds](#csi-workarounds).
+      ```
+      application_node_config.yaml
+      cabinets.yaml
+      hmn_connections.json
+      ncn_metadata.csv
+      switch_metadata.csv
+      ```
 
-   1. If doing a first time install or the `system_config.yaml` parameter file for a reinstall is not available, generate the system configuration.
+   1. Set an environment variable so this system name can be used in later commands.
 
-      If doing a first time install, this step is required. If you did the previous step as part of a reinstall, skip this.
+      ```bash
+      pit# export SYSTEM_NAME=eniac
+      ```
 
-      1. Check for the configuration files. The needed files should be in the current directory.
+   1. Generate the system config:
+      > **`NOTE`** the provided command below is an **example only**, run `csi config init --help` to print a full list of parameters that must be set. These will vary sifnificatnly depending on ones system and site configuration.
 
-         ```bash
-         linux# ls -1
-         ```
+      ```bash
+      pit# csi config init \
+          --bootstrap-ncn-bmc-user root \
+          --bootstrap-ncn-bmc-pass changeme \
+          --system-name ${SYSTEM_NAME} \
+          --can-cidr 10.103.11.0/24 \
+          --can-external-dns 10.103.11.113 \
+          --can-gateway 10.103.11.1 \
+          --can-static-pool 10.103.11.112/28 \
+          --can-dynamic-pool 10.103.11.128/25 \
+          --nmn-cidr 10.252.0.0/17 \
+          --hmn-cidr 10.254.0.0/17 \
+          --ntp-pools time.nist.gov \
+          --site-domain dev.cray.com \
+          --site-ip 172.30.53.79/20 \
+          --site-gw 172.30.48.1 \
+          --site-nic p1p2 \
+          --site-dns 172.30.84.40 \
+          --install-ncn-bond-members p1p1,p10p1 \
+          --application-node-config-yaml application_node_config.yaml \
+          --cabinets-yaml cabinets.yaml \
+          --hmn-mtn-cidr 10.104.0.0/17 \
+          --nmn-mtn-cidr 10.100.0.0/17 \
+          --bgp-peers aggregation
```

Comment:
looks like we're missing some new options here?  e.g. for the CMN.

Reply:
Correct, I added a notice to use the help function since I didn't have _this_ formatted & filled-in example handy. Since you commented though, I'll update this to current.